.github/workflows/build.yml

```diff
@@ -5,25 +5,41 @@ on:
   push:
     branches:
       - master
+      - release/v*
```

Comment:
We require build checks to pass when merging to a release branch but we weren't running the checks.